logstash-core/src/test/java/org/logstash/ackedqueue/TestSettings.java

```diff
@@ -9,7 +9,7 @@
 
 public class TestSettings {
 
-    public static Settings getSettings(int capacity) {
+    public static Settings volatileQueueSettings(int capacity) {
```

Comment:
because the `TestSettings` class had `getSettings()` and `getSettingsCheckpointFilePageMemory()` which has 2 problems:
1- since `TestSettings` now exposes both volatile and persisted settings, `getSettings()` means nothing.
2- I have not idea what `getSettingsCheckpointFilePageMemory()` even mean